ptu/global.ini

```diff
@@ -68954,81 +68954,81 @@ usable_stop=Stopp
-vaughn_assassination_E_desc_intro=Du hast dich in letzter Zeit so gut geschlagen, dass ich sehen möchte, ob du nicht zu mehr fähig bist.\n\nDieses Ziel wird etwas schwieriger sein als das, was du bisher übernommen hast, aber ich erwarte dennoch, dass es eine ziemlich klare Angelegenheit wird.\n\nDein Ziel ist ~mission(TargetName). Sie werden wahrscheinlich in der Nähe von ~mission(Location|Address) mit einem Begleitschutz oder zwei reisen, und aufgrund ihrer Verbindungen besteht eine gute Chance, dass sie versiertere Piloten sind als der Durchschnitt.\n\nLass uns sehen, was du tun kannst, wenn die Chancen gegen dich stehen.\n\nMit freundlichen Grüßen,\nVaughn\n
-vaughn_assassination_E_title_001=A Straightforward Task
-vaughn_assassination_E_title_002=An Uncomplicated Task
-vaughn_assassination_E_title_003=A Basic Task
+vahicle_DescAEGS_Nautilus=Hersteller: Aegis Dynamics\nFokus: Minelayer\n\nMit vier Jahrhunderten herausragender Dienste auf dem Buckel stellt der taktische Minenleger Nautilus von Aegis Dynamics das Nonplusultra in der strategischen Kampftechnik dar, mit Vollständig integrierten Mineneinsatz-, -räum- und -entschärfungsfähigkeiten.
+vaughn_assassination_E_desc_001=~mission(TargetName) muss ausgeschaltet werden. Meine Kontakte sagen mir, dass er ein oder zwei Begleitschützer haben wird, aber nichts, was Sie nicht bewältigen können. Sie werden ihn in der Nähe von ~mission(Location|Address) finden.\n\nMit freundlichen Grüßen,\nVaughn\n
+vaughn_assassination_E_desc_002=Noch eine Aufgabe für Sie. ~mission(TargetName). Der aktuelle Standort wird vermutlich ~mission(Location|Address) sein.\n\nIch vertraue darauf, dass Sie ihn ausschalten und problemlos mit der Eskorte fertig werden, das er vielleicht aufgetrieben hat.\n\nMit freundlichen Grüßen,\nVaughn\n
+vaughn_assassination_E_desc_003=Ich benötige einen Verbündeten, um ~mission(TargetName) auszuschalten.\n\nIch erwarte jedoch nicht, dass er leise verschwindet, da er ein oder zwei Personen als Begleitschutz angeheuert hat.\n\nReisen Sie zu ~mission(Location|Address), kümmern Sie sich um die gesamte Angelegenheit und schalten Sie ~mission(TargetName) aus.\n\nMit freundlichen Grüßen,\nVaughn\n
+vaughn_assassination_E_desc_intro=Sie haben sich in letzter Zeit so gut geschlagen, dass ich sehen möchte, ob Sie nicht zu mehr fähig sind.\n\nDieses Ziel wird etwas schwieriger sein als das, was Sie bisher übernommen haben, aber ich erwarte dennoch, dass es eine ziemlich klare Angelegenheit wird.\n\nIhr Ziel ist ~mission(TargetName). Er wird wahrscheinlich in der Nähe von ~mission(Location|Address) mit Begleitschutz von ein oder zwei Personen reisen, und aufgrund seiner Verbindungen besteht eine gute Chance, dass es versiertere Piloten sind als der Durchschnitt.\n\nLass uns sehen, was du tun kannst, wenn die Chancen gegen dich stehen.\n\nMit freundlichen Grüßen,\nVaughn\n
+vaughn_assassination_E_title_001=Eine direkte Aufgabe
```

Comment:
Titel bitte nicht übersetzen. Von keinen Missionen. Weil es oft Probleme gab wenn deutsche mit Amis spielen

Reply:
Ausgenommen sind soweit ich gesehen habe _title_intro und _title_rehire

Reply:
Die gehören glaube nicht zum mobiglas zum Auftragsmanager. Im Auftragsmanager soll alles in eng bleiben vorerst